examples/basic_usage_examples.py

```diff
@@ -72,16 +72,16 @@
 ## (this may fail if the server doesn't support expand)
 print("Here is some icalendar data:")
 try:
-    events_fetched = my_new_calendar.date_search(
-        start=datetime(2021, 5, 16), end=datetime(2024, 1, 1), expand=True
+    events_fetched = my_new_calendar.search(
+        start=datetime(2021, 5, 16), end=datetime(2024, 1, 1),event= True, expand=True
```

Comment:
One space too much ... if you run `tox -e style`, it will autocorrect.  (I could do it, but better you do it to get it into the same commit in the master branch and with the correct annotation)